### ISSUE TITLE
[Ubuntu 20.04] Missing libs libcrypto.so.1.0.0 and libcrypto.so.1.0.0

### ISSUE THREAD

**Issue by jonrosner** [sam701/awscredx#16]
The two libraries are missing in newer versions of Ubuntu and therefore you cannot run `awscredx init`:
```bash
➜  ~ awscredx init
awscredx: error while loading shared libraries: libcrypto.so.1.0.0: cannot open shared object file: No such file or directory
```

 Linking them manually works but is nowhere documented.

**Comment by sam701** [comment]
Thanks for reporting. I will take a look.

**Comment by sam701** [comment]
Fixed in `0.9.1`. Now the linux binary is built on `ubuntu:20.04`.

@jonrosner You wrote you linked them manually. Could you share how exactly you did it?

**Comment by jonrosner** [comment]
I did a rather quick fix. The libraries are actually present from other sources. I don't know if this can be replicated on other systems

```bash
$ sudo ln -s /usr/lib/nsight-compute/host/linux-desktop-glibc_2_11_3-x64/libcrypto.so.1.0.0 libcrypto.so.1.0.0
$ sudo ln -s /usr/lib/nsight-compute/host/linux-desktop-glibc_2_11_3-x64/libssl.so.1.0.0 libssl.so.1.0.0
```
